src/unistd.rs

```diff
@@ -367,6 +391,15 @@ pub fn execve(path: &CString, args: &[CString], env: &[CString]) -> Result<Void>
     Err(Error::Sys(Errno::last()))
 }
 
+/// Replace the current process image with a new one and replicate shell `PATH`
+/// searching behavior (see
+/// [exec(3)](http://man7.org/linux/man-pages/man3/exec.3.html)).
+///
+/// See `::nix::unistd::execve` for additoinal details.  `execvp` behaves the
+/// sme as execv except that it will examine the `PATH` environment variables
+/// for file names not specified with a leading slash.  For example, `execv`
+/// would not work if I specified "bash" for the path argument, but `execvp`
```

Comment:
Sure, that can be accommodated.  In general, I am of the opinion that documentation changes moving forward should be fact-checked and merged first and improved in terms of content, quality, and consistency later, particularly given the somewhat poor state of our documentation at present.